xml/System.Net.Http/HttpClient.xml

```diff
@@ -574,8 +574,11 @@ The `DefaultRequestVersion` property can be changed as long as the <xref:System.
 > - <xref:System.Threading.Tasks.TaskCanceledException> that nests a <xref:System.TimeoutException> is thrown on .NET 5.0 and later versions.
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentNullException">The <paramref name="requestUri" /> is <see langword="null" />.</exception>
-        <exception cref="T:System.InvalidOperationException">The request message was already sent by the <see cref="T:System.Net.Http.HttpClient" /> instance.</exception>
+        <exception cref="T:System.InvalidOperationException">The request message was already sent by the <see cref="T:System.Net.Http.HttpClient" /> instance.
+
+-or-
+
+The <paramref name="requestUri" /> must be an absolute URI or <see cref="P:System.Net.Http.HttpClient.BaseAddress" /> must be set.</exception>
```

Comment:
```suggestion
The <paramref name="requestUri" /> is not an absolute URI.

-or-

<see cref="P:System.Net.Http.HttpClient.BaseAddress" /> is not set.</exception>
```

Reply:
I don't think only `is not an absolute URI` is a valid text, because `requestUri` can be a relative URI if a `BaseAddress` is set.